documentation/feature/backpacks.md

```diff
@@ -6,18 +6,11 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Lock backpack
-Adds the ability to lock backpacks. Locked backpacks can't be accessed by others.
+### 1.1 Zipping sounds
+When someone open your backpack you hear a zipping sound, this help figuring who steals your bananas.
```

Comment:
Bad grammar. How about: `A zipper sound rings out, if someone tries to open your backpack. This helps to prevent banana theft.`